### PR TITLE
Update configure-endpoints-sccm.md

### LINKED ISSUE #13121: Page titled "Onboard Windows devices using Configuration Manager" does not document steps to onboard Windows devices using Configuration Manager
The title of this page is, "Onboard Windows devices using Configuration Manager" but the page does not provide steps to onboard Windows devices using Configuration Manager.

These are the top-level headings on the page:
1. Prerequisites
2. Other recommended configuration settings
3. Offboard devices using Configuration Manager
4. Monitor device configuration

One might expect to find a heading titled "Onboard devices using Configuration Manager" with steps to do so, but no.

The steps are documented somewhere, but it is left to the reader to Google and hunt around the site to find the steps on another page:
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/onboarding-endpoint-configuration-manager

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: f609bde5-cd6a-a287-4aa9-f32eb81deb3d
* Version Independent ID: 0367d97f-d96a-ba47-9db0-d3f9b3e5091e
* Content: [Onboard Windows devices using Configuration Manager](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-endpoints-sccm?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @siosulli
* Microsoft Alias: **siosulli**

### PR DESCRIPTION
**Pull request by msbemba**
added steps to do  Onboarding using Microsoft Configuration Manager

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/13121

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md [hunk 1]
```diff
@@ -89,6 +89,59 @@ The default value in case the registry key doesn't exist is 1.
 
 For more information about System Center Configuration Manager Compliance, see [Introduction to compliance settings in System Center 2012 R2 Configuration Manager](/previous-versions/system-center/system-center-2012-R2/gg682139\(v=technet.10\)).
 
+### Onboard Windows devices using Microsoft Configuration Manager
+
+### Collection creation
+
+To onboard Windows devices with Microsoft Configuration Manager, the deployment can target an existing collection or a new collection can be created for testing.
+
+Onboarding using tools such as Group policy or manual method doesn't install any agent on the system.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md]:
```suggestion
Onboarding using tools such as Group Policy or a manual method doesn't install any agents on the system.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md [hunk 2]
```diff
@@ -89,6 +89,59 @@ The default value in case the registry key doesn't exist is 1.
...
+
+Within the Microsoft Configuration Manager, console the onboarding process will be configured as part of the compliance settings within the console.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md]:
```suggestion
Within the Microsoft Configuration Manager console, the onboarding process will be configured as part of the compliance settings within the console.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md [hunk 3]
```diff
@@ -89,6 +89,59 @@ The default value in case the registry key doesn't exist is 1.
...
+
+Any system that receives this required configuration maintains that configuration for as long as the Configuration Manager client continues to receive this policy from the management point.
+
+Follow the steps below to onboard endpoints using Microsoft Configuration Manager.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md]:
```suggestion
Follow these steps to onboard endpoints using Microsoft Configuration Manager:
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md [hunk 4]
```diff
@@ -89,6 +89,59 @@ The default value in case the registry key doesn't exist is 1.
...
+
+1. In Microsoft Configuration Manager console, navigate to **Assets and Compliance \> Overview \> Device Collections**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md]:
```suggestion
1. In the Microsoft Configuration Manager console, navigate to **Assets and Compliance \> Overview \> Device Collections**.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md [hunk 5]
```diff
@@ -89,6 +89,59 @@ The default value in case the registry key doesn't exist is 1.
...
+
+    :::image type="content" source="images/configmgr-device-collections.png" alt-text="The Microsoft Configuration Manager wizard1" lightbox="images/configmgr-device-collections.png":::
+
+2. Right select **Device Collection** and select **Create Device Collection**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md]:
```suggestion
2. Select and hold (or right-click) **Device Collection** and select **Create Device Collection**.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md [hunk 6]
```diff
@@ -89,6 +89,59 @@ The default value in case the registry key doesn't exist is 1.
...
+
+3. Provide a **Name** and **Limiting Collection**, then select **Next**.
+
+    :::image type="content" source="images/configmgr-limiting-collection.png" alt-text="The Microsoft Configuration Manager wizard3" lightbox="images/configmgr-limiting-collection.png":::
+
+4. Select **Add Rule** and choose **Query Rule**.
+
+    :::image type="content" source="images/configmgr-query-rule.png" alt-text="The Microsoft Configuration Manager wizard4" lightbox="images/configmgr-query-rule.png":::
+
+5. Select **Next** on the **Direct Membership Wizard** and select on **Edit Query Statement**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md]:
```suggestion
5. Select **Next** on the **Direct Membership Wizard** and then select **Edit Query Statement**.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md [hunk 7]
```diff
@@ -89,6 +89,59 @@ The default value in case the registry key doesn't exist is 1.
...
+
+    :::image type="content" source="images/configmgr-direct-membership.png" alt-text="The Microsoft Configuration Manager wizard5" lightbox="images/configmgr-direct-membership.png":::
+
+6. Select **Criteria** and then choose the star icon.
+
+    :::image type="content" source="images/configmgr-criteria.png" alt-text="The Microsoft Configuration Manager wizard6" lightbox="images/configmgr-criteria.png":::
+
+7. Keep criterion type as **simple value**, choose whereas **Operating System - build number**, operator as **is greater than or equal to** and value **14393** and select on **OK**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md]:
```suggestion
7. Keep criterion type as **simple value**, choose whereas **Operating System - build number**, operator as **is greater than or equal to** and value **14393**, and select **OK**.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md [hunk 8]
```diff
@@ -89,6 +89,59 @@ The default value in case the registry key doesn't exist is 1.
...
+8. Select **Next** and **Close**.
+
+    :::image type="content" source="images/configmgr-membership-rules.png" alt-text="The Microsoft Configuration Manager wizard8" lightbox="images/configmgr-membership-rules.png":::
+
+9. Select **Next**.
+
+    :::image type="content" source="images/configmgr-confirm.png" alt-text="The Microsoft Configuration Manager wizard9" lightbox="images/configmgr-confirm.png":::
+
+After completing this task, you now have a device collection with all the Windows endpoints in the environment.
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md]:
```suggestion
After completing this task you have a device collection with all the Windows endpoints in the environment.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**